python/packages/jumpstarter-driver-bt-peer/examples/demo_plan.md

```
@@ -0,0 +1,208 @@
+# Cuttlefish BT Testing Demo Plan
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)